lib/CONST.d.ts

```diff
@@ -0,0 +1,779 @@
+/**
+ * URL of our CloudFront Instance
+ */
+export declare const g_cloudFront: "https://d2k5nsl2zxldvw.cloudfront.net";
+/**
+ * URL of our image CDN
+ */
+export declare const g_cloudFrontImg: "https://d2k5nsl2zxldvw.cloudfront.net/images/";
+export declare const CONST: {
```

Comment:
@blazejkustra What about using [ReadonlyDeep](https://github.com/sindresorhus/type-fest/blob/main/source/readonly-deep.d.ts) from `type-fest`? In this way we don't need to type every property as `readonly`, making the code more readable.

Reply:
I prefer not to use type-fest for declaration files in expensify-common. This file was generated with `tsc`, I don't see advantages in making manual adjustments.